docusaurus/docs/cloud/advanced/database.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Database
+displayed_sidebar: cloudSidebar
+description: Configure your own database on Strapi Cloud.
+canonicalUrl: https://docs.strapi.io/cloud/advanced/database.html
+---
+
+# Database
+
+Strapi Cloud provides a pre-configured PostgreSQL database by default. However, you can also configure it to utilize an external SQL database, if needed.
+
+:::caution
+While supported, it is not recommended to use an external database with Strapi Cloud, unless there is an explicit need to do so. This is because Strapi Cloud provides a managed database that is optimized for Strapi. Using an external database may result in unexpected behavior and/or performance issues.
+
+### Caveats
+
+- Network latency may impact performance.
+- Strapi cannot provide security for external databases.
+- Strapi cannot provide support for external databases.
+
+:::
+
+:::prerequisites
+
+- A local Strapi project running on `v4.8.1+`.
+- Credentials for external database.
+- If using an existing database,
+- the schema must match the Strapi project schema.
+
+:::
+
+## Configuration
+
+It is important to ensure that the project `./config/database.js` or `./config/database.ts` file matches the configuration found in the [Environment variables in database configurations](https://docs.strapi.io/dev-docs/configurations/database#environment-variables-in-database-configurations) section.
+
+:::tip
```

Comment:
I wouldn't put that in a tip since this seems to be part of the typical procedure.
I suggest rewriting this and the following lines as follows:

> Before pushing changes, add environment variables to the Strapi Cloud project:
>
> 1. Log into Strapi Cloud and click on the corresponding project on the Projects page.
> 2. Click on the **Settings** tab and choose **Variables** in the left menu.
> 3. Add the following environment variables:
> _(add the table from lines 47-57 here — please don't forget to indent the whole table so it appears included in step 3 🙂)_
> 4. Click **Save**.